tests/test_expval.py

```diff
@@ -178,18 +183,25 @@ def test_hermitian_expectation(self, n_wires, theta, phi, qubit_device, tol):
             obs = qml.Hermitian(U, wires=perm)
 
             def circuit():
-                qml.StatePrep(init_state, wires=range(n_qubits))
+                if device_name != "lightning.tensor":
+                    qml.StatePrep(init_state, wires=range(n_qubits))
                 qml.RY(theta, wires=[0])
                 qml.RY(phi, wires=[1])
                 qml.CNOT(wires=[0, 1])
                 return qml.expval(obs)
 
             circ = qml.QNode(circuit, dev)
             circ_def = qml.QNode(circuit, dev_def)
-            assert np.allclose(circ(), circ_def(), tol)
+            if device_name != "lightning.tensor":
+                assert np.allclose(circ(), circ_def(), tol)
+            else:
+                assert np.allclose(circ(), circ_def(), rtol=0.25)
```

Comment:
Why do we need the special `tol`? Can't we get the exact answer with a large enough `max_bond` or low enough `cutoff`?

Reply:
Good question. It's something we need to explore/resolve it next quarter. The default `cutoff` is `0` already and `max_bond` is `128`. Havn't had a chance to explore but I guess the main issue is the `Hermitian` obs with 2+ wires.

Reply:
Yea, that rtol is very loose --- I think we should avoid such wide value ranges.

Reply:
some update. It works alright now. This was the addition I made when max_bond_dim was default as 10.